packages/gateway-client/src/worker/index.ts

```diff
@@ -560,17 +568,24 @@ async function main() {
         [];
     console.debug('got relay addrs', relay_addrs);
 
-    // update status
-    self.status.serverPeer = ServerPeerStatus.BOOTSTRAPPED;
-    self.status.relays.push(...relay_addrs);
-
     const { hostname } = new URL(self.origin);
     const [_, _proto, _ip, ...rest] = bootstrapaddr?.split('/') ?? [];
     const hostaddr = `/dns4/${hostname}/${rest.join('/')}`;
-    const bootstraplist = [bootstrapaddr ?? '', hostaddr, ...relay_addrs];
+    return [bootstrapaddr ?? '', hostaddr, ...relay_addrs];
+}
+
+async function main() {
+    // self.window.localStorage.debug = (await localforage.getItem('debug')) || ""
+
+    const bootstraplist = await getBootstrapList();
+    // update status
+    self.status.serverPeer = ServerPeerStatus.BOOTSTRAPPED;
+
+    self.status.relays.push(...bootstraplist.slice(2));
```

Comment:
Why do we only want to report the first two relays to the status object?

Reply:
the bootstrap list contains two records that aren't relays